### DISCUSSION TITLE
Media flags support

### DISCUSSION THREAD

**Discussion by iFredOS** [croneter/PlexKodiConnect#1776] [Q&A]
I want to know if you will add media flags (media infos) to pass through your add-on. I only see ??? or simply nothing. I tried different skins before I wrote this message.

Reference: https://kodi.wiki/view/Media_flags

**Comment by croneter** [comment]
That's already implemented and a basic functionality of PKC. E.g. using Confluence, see below. You need to be more specific, if something is missing, and share a screenshot or two. 

![image](https://user-images.githubusercontent.com/4637940/150493696-15d3591c-8a6e-40d2-a9fc-09910777b0ad.png)
![image](https://user-images.githubusercontent.com/4637940/150493805-89d96ca3-6c73-4102-888d-2ab82e50ae10.png)

**Reply by iFredOS** [reply]
I tried 3 different skin and I see nothing except ratings and the studios logo. I just downloaded the Confluence skin and tried every views and I only see if this is web or Blu-ray. Can you guide me how you did your setup for this to work? I cannot reproduce the setup I see in your screenshots.

Thanks!

**Reply by croneter** [reply]
Literally nothing special! Grabbed Kodi from www.kodi.tv (make sure you're not using some obscure other source). Installed PKC. Set it up to connect to my PMS. Done. 

Does the info/flags you're missing show up when you use Plex Web?!?
